airflow-core/src/airflow/api_fastapi/core_api/datamodels/plugins.py

```diff
@@ -65,20 +65,22 @@ class AppBuilderMenuItemResponse(BaseModel):
     model_config = ConfigDict(extra="allow")
 
     name: str
-    href: str | None = None
+    href: str
     category: str | None = None
 
 
-class IFrameViewsResponse(BaseModel):
+class ExternalViewResponse(BaseModel):
     """Serializer for IFrame Plugin responses."""
 
     model_config = ConfigDict(extra="allow")
 
     name: str
-    src: str
+    href: str
```

Comment:
`href` is required but only used if `url_route` is not used. Should we somehow make this clear that at least a placeholder value needs to be added?